tests/rbac_test.go

```diff
@@ -0,0 +1,212 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ktests "kubevirt.io/kubevirt-ansible/tests"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/log"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("RBAC", func() {
+	flag.Parse()
+
+	var vmi ktests.VMManifest
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	BeforeEach(func() {
+		By("Login with the user test_admin")
+		args := []string{"login", "-u", "test_admin", "-p", "123456"}
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		tests.BeforeTestCleanup()
+		vmi.Name = "vm-cirros"
+		vmi.Manifest = "tests/manifests/vm-cirros.yaml"
+		ktests.CreateUser()
+
+		By("Create a VM with yaml")
+		args = []string{"create", "-f", vmi.Manifest, "-n", tests.NamespaceTestAlternative}
+		_, err = tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Check VMI should create")
+		args = []string{"start", vmi.Name, "-n", tests.NamespaceTestAlternative}
+		_, err = tests.RunCommand("virtctl", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Check the vmi is running")
+		args = []string{"get", "vmi", vmi.Name, "--template", "{{.status.phase}}", "-n", tests.NamespaceTestAlternative}
+		Eventually(func() string {
+			output, err := tests.RunCommand("oc", args...)
+			Expect(err).ToNot(HaveOccurred())
+			return output
+		}, time.Minute*10).Should(Equal("Running"))
+	})
+
+	AfterEach(func() {
+		By("Login with the user test_admin")
+		args := []string{"login", "-u", "test_admin", "-p", "123456"}
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		ktests.DeleteUser()
+
+		By("Delete identity of user kubevirt-test-user")
+		args = []string{"delete", "identity", "allow_all_auth:kubevirt-test-user"}
+		_, err = tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	DescribeTable("RBAC Authorization", func(role, namespace string, con bool) {
+		By(fmt.Sprintf("Add %s to user kubevirt-test-user", role))
+		args := []string{"adm", "policy", "add-role-to-user", role, ktests.UsernameTestUser, "-n", namespace}
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Login with the user kubevirt-test-user")
+		args = []string{"login", "-u", ktests.UsernameTestUser, "-p", "123456"}
```

Comment:
Again, test user info should also be passed though environment variables

Reply:
This can be skipped for this PR, as the change is a wider discussion for all tests